invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/index.js

```diff
@@ -0,0 +1,7 @@
+// This file is part of InvenioAdministration
```

Comment:
```suggestion
// This file is part of Invenio-RDM-Records
```

There are other occurrences of this in the PR